uint64 timestamp
bool safety_switch_available		# Set to true if a safety switch is connected
bool safety_off			# Set to true if safety is off
